hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -63,6 +64,9 @@ public void migrate(@NonNull final MigrationContext ctx) {
                 ctx.newStates().get(NODES_KEY);
         final var networkInfo = ctx.networkInfo();
         final var addressBook = networkInfo.addressBook();
+        final var bootstrapConfig = ctx.configuration().getConfigData(BootstrapConfig.class);
+        final var adminKey =
```

Comment:
Are there E2E tests added for adding `adminKey` in `NodeCreate` and `NodeUpdateHandler`. Each PR should have E2E tests corresponding to the change

Reply:
We are going to work on E2E test after this.